### PR TITLE
Prepared App for different localisations

### PR DESCRIPTION
**Pull request by sauberfred**
Switched from hardcoded strings to NSLocalizedStrings

### CODE REVIEW

File: Icro/ViewController/SettingsViewController.swift [hunk 1]
```diff
@@ -13,6 +13,7 @@ class SettingsViewController: UIViewController {
     @IBOutlet weak var outterStackView: UIStackView!
     @IBOutlet private weak var blogSetupView: UIView!
     @IBOutlet private weak var blogSetupSwitch: UISwitch!
+    @IBOutlet weak var blogSetupSwitchLabel: UILabel!
```
Comment (hartlco) [inline Icro/ViewController/SettingsViewController.swift]: I think all the newly introduced outlets should be `private` properties as well.

**Review by hartlco — CHANGES_REQUESTED** [review]
This is a great PR, thank you!
I noticed this PR introduces 4 new warning because of SwiftLint. Those should be fixed before merging. For the future I plan to integrate the SwiftLint checks into the CI pipeline.

I also added one comment about the outlets in the Settings.

Let me know if you are interested in fixing the issues, otherwise I would merge it and fix it in hindsight!

**Comment by sauberfred** [comment]
You are welcome!
I fixed the SwiftLint warnings and made the new outlets private.

File: Icro/ViewController/InsertLinkViewController.swift [hunk 2]
```diff
@@ -10,8 +10,8 @@ class InsertLinkViewController: UIViewController, UITextFieldDelegate {
 
     private var cancelButton: UIBarButtonItem?
     @IBOutlet private weak var titleTextField: UITextField!
-    @IBOutlet weak var linkTextField: UITextField!
-    @IBOutlet weak var insertButton: FakeTableCellButton!
+    @IBOutlet private weak var linkTextField: UITextField!
```
Comment (sauberfred) [inline Icro/ViewController/InsertLinkViewController.swift]: I also made linkTextField private, but can revert if you do not want it to be private.

**Comment by hartlco** [comment]
Great, merging now!

**Review by hartlco — APPROVED** [review]

**Pull request merged**